part-spec/ic_micro.json

```diff
@@ -0,0 +1,76 @@
+{
+    "$id": "https://github.com/chromeos/digital-datasheets/blob/main/part-spec/ic micro.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "ic micro",
+    "oneOf": [
+        {
+            "title": "microcontroller/ec",
+            "type": "object",
+            "required": [
+                "componentID"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "onChipFlash": {
+                    "description": "quantity of built-in flash in a microprocessor",
+                    "comment": "units of kilobytes",
```

Comment:
I'm a bit confused at how this can be in "units of kilobytes" when the "unit" type only has bytes.

I'm also a bit confused at why you need a "unit" type here when there is only exactly one correct option for this field - if we want this field to be kilobytes, shouldn't we force the unit to be kilobytes somehow?

Reply:
You are correct that this comment isn't really enforceable, but I actually think that's okay given the way the "units" field works. The units field is explicit about exactly what the multiplier on the base unit is. This is really meant more as a human-readable hint for someone reading the spec and less about what this actually needs to be.

Reply:
Kevin - Can we close this open or do you need more changes?

Reply:
Sorry for the delay!